pkg/local_object_storage/shard/get.go

```diff
@@ -94,24 +94,24 @@ func (s *Shard) Get(prm GetPrm) (GetRes, error) {
 // fetchObjectData looks through writeCache and blobStor to find object.
 func (s *Shard) fetchObjectData(addr oid.Address, skipMeta bool, cb storFetcher, wc func(w writecache.Cache) (*objectSDK.Object, error)) (*objectSDK.Object, bool, error) {
 	var (
-		err error
-		res *objectSDK.Object
+		mErr error
+		mRes meta.ExistsRes
 	)
 
 	var exists bool
 	if !skipMeta {
 		var mPrm meta.ExistsPrm
 		mPrm.SetAddress(addr)
 
-		mRes, err := s.metaBase.Exists(mPrm)
-		if err != nil && !s.info.Mode.NoMetabase() {
-			return res, false, err
+		mRes, mErr = s.metaBase.Exists(mPrm)
```

Comment:
Could you explain what the problem was in the commit message?

Reply:
reworded commit message